components/dss-tools/plugins/org.wso2.integrationstudio.artifact.dataservice/src/org/wso2/integrationstudio/artifact/dataservice/ui/wizard/GenerateDataServicesUtils.java

```diff
@@ -0,0 +1,270 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.integrationstudio.artifact.dataservice.ui.wizard;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class GenerateDataServicesUtils {
+    
+    
+    private static Map<Integer, String> definedTypeMap = new HashMap<Integer, String>();
+
+    private static Map<Integer, String> qnameTypeMap = new HashMap<Integer, String>();
+    
+    public enum Methods
+    {
+        GET , POST , PUT , DELETE;
+    }
+
+    static {
+        definedTypeMap.put(java.sql.Types.CHAR, "STRING");
+        definedTypeMap.put(java.sql.Types.NUMERIC, "NUMERIC");
+        definedTypeMap.put(java.sql.Types.DECIMAL, "DOUBLE");
+        definedTypeMap.put(java.sql.Types.INTEGER, "INTEGER");
+        definedTypeMap.put(java.sql.Types.SMALLINT, "SMALLINT");
+        definedTypeMap.put(java.sql.Types.FLOAT, "DOUBLE");
+        definedTypeMap.put(java.sql.Types.REAL, "REAL");
+        definedTypeMap.put(java.sql.Types.DOUBLE, "DOUBLE");
+        definedTypeMap.put(java.sql.Types.VARCHAR, "STRING");
+                definedTypeMap.put(java.sql.Types.NVARCHAR, "STRING");
+        // typeMap.put(java.sql.Types.LONGNVARCHAR, "STRING");
```

Comment:
Shall we remove these commented lines?

Reply:
Done https://github.com/wso2/integration-studio/pull/890/commits/7c67b03c80855728fbecedbfaad2e6f19af7f2c6